[DOC] Suggest removal of unused fields from Kafka CR after migration to node pools (#9809)
strimzi/strimzi-kafka-operator: documentation/modules/configuring/proc-migrating-clusters-node-pools.adoc

----
+
There is no change or rolling update.
The resources remain identical to how they were before.

> . Once the `KafkaNodePool` resource is in use, you can remove the old fields from the `Kafka` custom resource.
> This includes all the fields that you copied to the `KafkaNodePool` resource in the previous steps such as the `.spec.kafka.replicas` or `.spec.kafka.storage`.

Reviewer: ```suggestion
. Remove the replicated properties from the `Kafka` custom resource.
When the `KafkaNodePool` resource is in use, you can remove the properties that you copied to the `KafkaNodePool` resource, such as the `.spec.kafka.replicas` and `.spec.kafka.storage` properties.
```
